doc/usermacros.rst

```diff
@@ -6,7 +6,7 @@ User macro reference
    :param DIRECTORY: Directory containing messages.  May be absolute or
      relative to ``CMAKE_CURRENT_SOURCE_DIR``.
    :param FILES:  Files containing messages, relative to `msgdir`
-   :param PACKAGE:  Optional alternate packagename (if the current project doesn't match the
+   :param BASE_DIR:  Optional alternate packagename (if the current project doesn't match the
```

Comment:
The user should probably never use it. It is only used by actions internally: https://github.com/ros/genmsg/blob/indigo-devel/cmake/genmsg-extras.cmake.em#L90-L93